src/plugins/data_source_management/public/types.ts

```diff
@@ -50,21 +50,15 @@ export type DataSourceManagementContextValue = OpenSearchDashboardsReactContextV
   DataSourceManagementContext
 >;
 
-export interface UpdatePasswordFormType {
```

Comment:
why remove this, I think it's still used somewhere in the update password flow?

Reply:
I thought that we would not have old password field or confirm new password field. That's why I choose to remove this. But I will add it back for now. Manny is working on editing page, he may have different ways to deal with that.

Reply:
yeah, remove it for now, that's possibly causing your build CI to fail.